lib/std/macho.zig

```diff
@@ -268,6 +268,343 @@ pub const nlist_64 = extern struct {
     n_value: u64,
 };
 
+/// Defines the attributes of a LC_TWOLEVEL_HINTS load command
+pub const twolevel_hints_command = struct { 
```

Comment:
I know existing things in this file aren't; but should these be TitleCase to follow zig conventions?